app.vup.Vup.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: app.vup.Vup
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory : "lib/ffmpeg"
+    version: "21.08"
+    add-ld-path: .  
+cleanup-commands:
+- mkdir -p ${FLATPAK_DEST}/lib/ffmpeg      
+command: vup
+separate-locales: false
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--device=dri"
+- "--socket=pulseaudio"
+- "--share=network"
+- "--filesystem=home"
+- "--talk-name=org.kde.StatusNotifierWatcher"
+- "--talk-name=org.freedesktop.Notifications"
+- "--filesystem=xdg-run/app/com.discordapp.Discord:create"
+modules:
+- name: libjsoncpp
+  buildsystem: cmake
+  config_opts:
+  - -DCMAKE_BUILD_TYPE=release
+  - -DBUILD_SHARED_LIBS=ON
+  sources:
+  - type: archive
+    url: https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.7.5.tar.gz
+    sha256: 4338c6cab8af8dee6cdfd54e6218bd0533785f552c6162bb083f8dd28bf8fbbe
+- name: libsodium
+  buildsystem: autotools
+  sources:
+  - type: archive
+    url: https://github.com/jedisct1/libsodium/archive/refs/tags/1.0.18-RELEASE.tar.gz
+    sha256: b7292dd1da67a049c8e78415cd498ec138d194cfdb302e716b08d26b80fecc10
+- name: intltool
```

Comment:
please use the shared-modules for this. https://github.com/flathub/shared-modules/